meinberlin/assets/scss/_variables.scss

```diff
@@ -86,3 +86,5 @@ $input-padding-x: 0.6rem !default;
 
 $form-select-border-radius: (0.3 * $spacer) !default;
 
+// bootstrap modal overwrites
+$modal-content-border-radius: 0;
```

Comment:
you need to add `!default`when setting variables, like we do with the others, also it's not overwrites when we are setting the variables, it's just modal variables or something